pkg/console/starter/starter.go

```diff
@@ -161,17 +161,30 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	versionRecorder.SetVersion("operator", os.Getenv("RELEASE_VERSION"))
 
 	clusterOperatorStatus := status.NewClusterOperatorStatusController(
-		"console",
+		// name
+		api.OpenShiftConsoleName,
+		// related objects
 		[]configv1.ObjectReference{
-			{Group: "operator.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "config.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "oauth.openshift.io", Resource: "oauthclients", Name: api.OAuthClientName},
-			{Resource: "namespaces", Name: api.OpenShiftConsoleOperatorNamespace},
-			{Resource: "namespaces", Name: api.OpenShiftConsoleNamespace},
+			// top level configs
+			{Group: configv1.GroupName, Resource: "consoles", Name: api.ConfigResourceName},
+			{Group: configv1.GroupName, Resource: "infrastructures", Name: api.ConfigResourceName},
```

Comment:
Not sure whether infrastructure config should be listed this in our related objects even though we watch it. This might imply a closer relationship than exists here. Is there a rule of thumb for what to include?

Reply:
I believe if we need it for state/building our config, we should include it.